library_generation/utils/utilities.py

```diff
@@ -224,7 +228,10 @@ def generate_postprocessing_prerequisite_files(
     # is one of grpc, http and both,
     if transport == "grpc":
         converted_transport = "grpc"
-    elif transport == "rest":
+    elif transport in [
```

Comment:
I guess this is because we use `rest` in BUILD but `http` in `repo-metadata.json`? This is convenient but I feel we should just stick with `rest` for consistency and simplicity.

Reply:
Done.